src/component/ConsentManagementPlatform.tsx

```diff
@@ -130,8 +143,8 @@ interface Props {
     onClose: () => void;
     source?: string;
     variant?: string;
+    fontFamilies?: FontsContextInterface;
```

Comment:
These 3 items are optional but some will be used more often than others so I'd suggest ordering by most likely to be used to least likely to be used. So it would be fontFamilies, source, variant. What do you think?

Reply:
I'm going to leave this for now as the ordering doesn't make a difference to the execution and the frequency of use is open to change.

Reply:
I'm okay with that. 👍